Makefile

```diff
@@ -17,13 +17,13 @@ EXTRA_CFLAGS=
 EXTRA_CXXFLAGS=
 
 # Set to 1 to enable hot/cold linking
-USE_PACKAGE:=0
+USE_PACKAGE:=1
 
 # Set this to 1 to add additional rules to compile your project as a PROS library template
 IS_LIBRARY:=1
 # TODO: CHANGE THIS!
 LIBNAME:=gif-pros
-VERSION:=2.1.1
```

Comment:
That's not a logical semantic version bump. I'd say 3.0.0 is correct.

Reply:
The rationale behind the version number is that the 8 stands for LVGL version 8. If that doesn’t make sense to you I’ll change it.

Reply:
Yeah, usually libraries don't inherit version numbers from their dependencies.